### ISSUE TITLE
Could you share the SKU110K-R dataset?

### ISSUE THREAD

**Issue by qjadud1994** [Anymake/DRN_CVPR2020#3]
Hello.
I want to conduct an experiment on the SKU110K-R dataset.

Could you share the SKU100K-R dataset which contains annotations for oriented bounding box?

**Comment by Panxjia** [comment]
@qjadud1994 Sure, it may take two or three days.  The entire dataset is very large, about 200G. We are trying to share it in an efficient way.

**Comment by edwardnguyen1705** [comment]
@Panxjia ,
Thanks so much. I am looking forward!

**Comment by Panxjia** [comment]
@qjadud1994 Sorry for the delay. We found the author of SKU110K released a revised version of data set. Specifically, compared with the previous version, there are 14 (8233Vs8219) fewer images in the training set and 5(2941Vs2936) images in the test set. We revised the annotations and the augment script accordingly.  Another thing to note is that in the author's newly released data set, some pictures may not be completed(train_5822, train_882, train_4222, train_924, test_274), the errors may be reported during using(cv2.imread, PIL.Image.open). Due to the permissions issues, we can only release the rotation augment script and the annotations of SKU110K-R. Both will be uploaded today.

**Comment by qjadud1994** [comment]
Thank you very much for your kind answer.

That would help me a lot.

Thank you again.

**Comment by edwardnguyen1705** [comment]
@Panxjia 
Thank you for sharing your work. We really appreciate your time and effort.

**Comment by zhangyuxuann** [comment]
> @qjadud1994 Sorry for the delay. We found the author of SKU110K released a revised version of data set. Specifically, compared with the previous version, there are 14 (8233Vs8219) fewer images in the training set and 5(2941Vs2936) images in the test set. We revised the annotations and the augment script accordingly. Another thing to note is that in the author's newly released data set, some pictures may not be completed(train_5822, train_882, train_4222, train_924, test_274), the errors may be reported during using(cv2.imread, PIL.Image.open). Due to the permissions issues, we can only release the rotation augment script and the annotations of SKU110K-R. Both will be uploaded today.

it seems the annotations for the test set are still 2941.
